client/tailwind.config.ts

```diff
@@ -1,24 +1,141 @@
 import type { Config } from 'tailwindcss'
 
-const config: Config = {
+export default {
+  darkMode: ['class'],
   content: [
     './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
     './src/components/**/*.{js,ts,jsx,tsx,mdx}',
     './src/app/**/*.{js,ts,jsx,tsx,mdx}',
+    './node_modules/@wallet-ui/react/dist/**/*.{js,ts,jsx,tsx}',
```

Comment:
You should be able to use `@import "@wallet-ui/tailwind/index.css";` in your global CSS and get rid of these 2 lines here.